### ISSUE TITLE
Deploy previews (trading): missing env vars, browser wallet fails

### ISSUE THREAD

**Issue by edd** [vegaprotocol/frontend-monorepo#6986]
## Description

After #6942 we have more useful deploy previews on PRs, but there are still a few things that aren't correct:

- [x] ~APP_NAME not set correctly in title~ this is set in `constants.ts`, not actually in from the env_var any more
  - <img width="251" alt="Screenshot 2024-09-23 at 10 30 07" src="https://github.com/user-attachments/assets/0abf1d61-ff3c-46db-87df-e3514b70bf6f">
- [ ] Creating a new browser wallet doesn't work

**Comment by edd** [comment]
Still in progress, trying to trace back to why 
- [x] `nx-cache` doesn't appear to be the issue
- [x] `yarn install --pure-lockfile` doesn't appear to be the issue
- [x] `nvm` doesn't appear to trigger it
- [x] clearing github caches doesn't help

Some build step appears to be switching whether or not [@vegaprotocol/js-crypto](https://github.com/vegaprotocol/js-crypto) is using the browserify-ed version or the alternative, which expects node:crypto, and fails when it isn't there. But this is only happening to deploy previews - dist (fairground, anywhere else) and local builds don't have the same issue

**Comment by JonRay15** [comment]
No longer relevant
